client/js/components/procedure/admin/DpAddOrganisationList.vue

```diff
@@ -147,11 +230,57 @@ export default {
         .catch(() => {
           dplan.notify.error(Translator.trans('warning.invitable_institution.not.added'))
         })
+    },
+
+    getInstitutionsWithContacts () {
+      const invitableToebFields = [
+        'legalName',
+        'participationFeedbackEmailAddress',
+        'locationContacts'
+      ]
+
+      const locationContactFields = ['street', 'postalcode', 'city']
+
+      if (hasPermission('field_organisation_email2_cc')) {
+        invitableToebFields.push('ccEmailAddresses')
+      }
+
+      if (hasPermission('field_organisation_contact_person')) {
+        invitableToebFields.push('contactPerson')
+      }
+
+      if (hasPermission('field_organisation_competence')) {
+        invitableToebFields.push('competenceDescription')
+      }
+
+      if (hasPermission('field_organisation_phone')) {
+        locationContactFields.push('phone')
+      }
```

Comment:
I would extract this into its own function:
```suggestion
     returnArrayofValues (permissionChecks) {
      return permissionChecks
        .filter(check => hasPermission(check.permission) !== true)
        .map(check => check.value)
    },
```
that way we could easily add permission check (or remove them) like that:

```suggestion
      const permissionChecksToeb = [
        { permission: 'field_organisation_email2_cc', value: 'ccEmailAddresses' },
        { permission: 'field_organisation_contact_person', value: 'contactPerson' },
        { permission: 'field_organisation_competence', value: 'competenceDescription'}
      ]

      const permissionChecksContact = [
        { permission: 'field_organisation_phone', value: 'phone' }
      ]
      
      this.invitableToebFields.concat(this.returnArrayofValues(permissionChecksToeb))
      this.locationContactFields.concat(this.returnArrayofValues(permissionChecksContact))
      
```